src/signals/incident-management/containers/DashboardContainer/components/StatusChart/index.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { PieChart, Pie, Cell, Tooltip, Legend } from 'recharts';
+
+import './style.scss';
+
+const RADIAN = Math.PI / 180;
+
+function renderStatusLabel({ cx, cy, midAngle, innerRadius, outerRadius, percent, index }) {
+  const percentage = (percent * 100).toFixed(0);
+  if (percentage === '0') {
+    return;
+  }
+ 	const radius = innerRadius + (outerRadius - innerRadius) * 0.5;
+  const x  = cx + radius * Math.cos(-midAngle * RADIAN);
+  const y = cy  + radius * Math.sin(-midAngle * RADIAN);
+  return (
+    <text className="status-label" x={x} y={y} fill="white" textAnchor="middle"	dominantBaseline="central">
+    	{`${percentage}%`}
+    </text>
+  );
+}
+
+const colors = {
+  'Gemeld': 'red',
+  'Afgehandeld': 'lightgreen',
+  'Heropend': 'orange',
+  'In afwachting van behandeling': 'purple',
+  'In behandeling': 'blue',
+  'On hold' : 'grey',
+  'Geannuleerd' : 'darkgrey'
```

Comment:
Better to use the defined constants for the statuses (statusList.js)

Reply:
veel beter gedaan